OverlayPlugin.Core/EventSources/MiniParseEventSource.cs

```diff
@@ -265,6 +338,10 @@ private void LogLineHandler(bool isImport, LogLineEventArgs args)
                 return;
             }
 
+            foreach (var kvp in LogLineMatchers)
+                if (kvp.Value.Value == null && Regex.IsMatch(args.originalLogLine, kvp.Value.Key, RegexOptions.Compiled))
+                    LogLineMatchers[kvp.Key]=new KeyValuePair<string, string>(kvp.Value.Key, args.originalLogLine);
+
```

Comment:
You can see further down that any line that's processed by `LogLineHandler` is also dispatched as an event. You're basically dispatching all the log lines matching the regex a second time. If the overlay doesn't subscribe to the log line `LogLine` event, this could save traffic. However, implementing that would be better solved by extending the event filter in `EventDispatcher` and allow overlays to specify a regex when they subscribe to the `LogLine` event.
Saving a bit of network traffic isn't worth the added complexity and overhead, though.